packages/react/src/components/Authenticator/SignIn/SignIn.tsx

```diff
@@ -22,8 +28,9 @@ export function SignIn() {
   };
 
   return (
-    // TODO Automatically add these namespaces again from `useAmplify`
-    <>
+    <View>
+      <Header />
```

Comment:
Notice how `SignIn.Header` and `SignIn.Footer` are **outside** the `<Form>`!

`FormFields` should be sufficient for re-use to prepend/append.